src/common/options.cc

```diff
@@ -7491,6 +7491,10 @@ std::vector<Option> get_mds_options() {
     .set_flag(Option::FLAG_NO_MON_UPDATE)
     .set_description("path to MDS data and keyring"),
 
+    Option("mds_fs", Option::TYPE_STR, Option::LEVEL_ADVANCED)
+    .set_default("")
```

Comment:
With the change to have beacons update `standby_daemon_fscid`, this could add RUNTIME flag.